docker/diffusers-flax-cpu/Dockerfile

```diff
@@ -43,6 +43,9 @@ RUN python3 -m pip install --no-cache-dir --upgrade pip uv==0.1.11 && \
         numpy==1.26.4 \
         scipy \
         tensorboard \
-        transformers
+        transformers \
+        hf_transfer
```

Comment:
I think this is not the way. From the doc:

> e.g.  pip install huggingface_hub[hf_transfer]

Reply:
Both are fine yes :+1: 
Using `huggingface_hub[hf_transfer]` is less risky as it delegates the potential version issues to hfh (see [here](https://github.com/huggingface/huggingface_hub/blob/c0fd4e0f7519a4e3659c836081cc7e38c0d14b35/setup.py#L40)) but for this specific case it's more than fine to just install hf_transfer (especially in a CI)